intel_extension_for_transformers/transformers/utils/config.py

```diff
@@ -891,6 +891,7 @@ def __init__(
         self.calib_func = kwargs.get("calib_func", None)
         self.calib_iters = kwargs.get("calib_iters", 100)
         self.scheme = "asym" if self.zero_point else "sym"
+        self.sym = not self.zero_point

```

Comment:
if it have zero point, but all value is 0, it's still sym right?

Reply:
fixed